prac_05/word_occurrences.py

```diff
@@ -0,0 +1,26 @@
+"""
+Count word occurrences in a string, sort alphabetically case-insensitive.
+
+Estimate: < 1 hour
+Actual:   1.5 hours
+"""
+
+word_occurrences = {}
+
+string = input("Enter text: ").lower()
+string = list(string.split())
+
+for word in string:
```

Comment:
Consider using the EAFP (Easier to Ask for Forgiveness than Permission) approach for counting words.